src/base/utils/misc.cpp

```diff
@@ -459,14 +459,15 @@ QString Utils::Misc::libtorrentVersionString()
     return version;
 }
 
-QString Utils::Misc::opensslVersionString()
+QString Utils::Misc::sslVersionString()
 {
 #if (OPENSSL_VERSION_NUMBER >= 0x1010000f)
-    static const auto version {QString::fromLatin1(OpenSSL_version(OPENSSL_VERSION))};
+    // OPENSSL_VERSION_TEXT expands to LIBRESSL_VERSION_TEXT when used with LibreSSL
+    static const auto version {QString::fromLatin1(OPENSSL_VERSION_TEXT)};
```

Comment:
It was deliberately changed to report run time version not the compile time one.

Reply:
Oh yeah - that's probably a good idea. I'll try get something that reports for run time